Purchases/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -155,6 +155,10 @@ internal protocol StoreProductType {
     var introductoryDiscount: StoreProductDiscount? { get }
 
     /// An array of subscription offers available for the auto-renewable subscription.
+    /// - Note: the currenet user may or may not be eligible for some of these.
+    /// - Seealso: ``Purchases/checkPromotionalDiscountEligibility(forProductDiscount:product:)``
+    /// - Seealso: ``Purchases/checkPromotionalDiscountEligibility(forProductDiscount:product:completion:)``
+    /// - Seealso: ``StoreProduct/getEligibleDiscounts()``
```

Comment:
we have a new format for related symbols now with #1253, but we can update the docs here after the PR is merged so we don't hold this one any longer

Reply:
Changed! Could you take another look to make sure I didn't mess up the new format?